src/cargo/lib.rs

```diff
@@ -30,6 +32,8 @@ extern crate url;
 
 use std::io;
 use std::fmt;
+use std::error::Error;
+use error_chain::ChainedError;
```

Comment:
While you're at it mind throwing a newline above this import of `error_chain` to separate the std and non-std imports?